src/pyfmi/fmi.pyx

```diff
@@ -30,6 +30,7 @@ import fnmatch
 import re
 from collections import OrderedDict
 cimport cython
+from io import StringIO, TextIOWrapper, UnsupportedOperation
```

Comment:
Doesn't seem like we need these?

Reply:
Correct, I've removed the first 2 (we still need the UnsupportedOperation)